app/src/main/java/one/mixin/android/db/PinMessageDao.kt

```diff
@@ -0,0 +1,81 @@
+package one.mixin.android.db
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Query
+import androidx.room.RoomWarnings
+import one.mixin.android.vo.ChatHistoryMessageItem
+import one.mixin.android.vo.PinMessage
+import one.mixin.android.vo.PinMessageItem
+import one.mixin.android.vo.PinMessageMinimal
+
+@Dao
+interface PinMessageDao : BaseDao<PinMessage> {
+    @Query("DELETE FROM pin_messages WHERE message_id IN (:messageIds)")
```

Comment:
If have 1000 message ids, is it ok?

Reply:
When sending with [chunked](https://github.com/MixinNetwork/android-app/commit/bd27038f1ad03db1868d876aa661a5dfb825baac#diff-f5357d4d283c8186ebaf4fe717962ecd97d24163abc3b2251b01fc03ea123564R144) 128, when receiving, the server limits the maximum pin or unpin size 128.